recipes/cdat_info/build.sh

```diff
@@ -0,0 +1,214 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC"
+export CXXLAGS="${CFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
+
+export GIT_DESCRIBE=`git describe --tags`
+
+cat > cdat_info.py.in << EOF
```

Comment:
hum all of our packages use some parts of this, uv-cdat used to be a big monolithic code and we split it in smaller more or less independent pieces now. But cdat_info is sort of a reference for all pacakges to know what's going on. for example they all use pingDB/get_sample_data_path, etc.. function to send (if user approved of course) anonymous usage info to help us improve the software. 
I'm not sure what your issue against the package is? Is it because it does not really have any reason to be as a stand-alone?